### PR TITLE
Control pause

### PR DESCRIPTION
**Pull request by curlydingo**
Per #218, hopefully better showers for all now :)

I've taken the liberty of incrementing the minor version number, but I'm not sure if that's appropriate or should happen later somehow. Apologies if I got it wrong.

### CODE REVIEW

**Comment by curlydingo** [comment]
Hmm, the linter doesn't seem to like that the new PauseEndTimeSensor has 13 lines of code that match the RainDelayStopTimeSensor.

I could:
1. Make a new class "EndTimeSensor" and derive PauseEndTimeSensor and RainDelayStopTimeSensor from it (seems like overkill), or
2. Change a comment or something small in the middle of that block (seems like a bit of a kludge).

Or another option you might come up with? Happy either way just let me know.

**Comment by vinteo** [comment]
Maybe a different icon (might make sense anyway) might help, but not sure if it be enough

Also I didn't give a minor verion for the other service and usually don't for dep upgrade so a patch version might be ok

**Comment by vinteo** [comment]
looks like I forgot to update the vsersion there so it should be 1.2.7 for a patch

File: custom_components/opensprinkler/sensor.py [hunk 1]
```diff
@@ -240,6 +241,48 @@ def _get_state(self):
...
+        super().__init__(entry, name, coordinator)
+
+    @property
+    def device_class(self):
+        """Return the device class."""
+        return SensorDeviceClass.TIMESTAMP
+
+    @property
+    def icon(self) -> str:
+        """Return icon."""
```
Comment (vinteo) [inline custom_components/opensprinkler/sensor.py:261]: I wonder if linting requires these comments, some of these seem pretty useless. Could help with the duplicate linting error if removed

Comment (vinteo) [reply custom_components/opensprinkler/sensor.py:261]: Another hack could be to just change the order of the properties... lol

**Review by vinteo — APPROVED** [review]

**Comment by curlydingo** [comment]
Sorry, only just saw the request for Ed's review! Don't release until he's had a look, I suppose?

**Comment by vinteo** [comment]
@curlydingo I forgot that we should add examples in the README. For the other service as well

**Comment by curlydingo** [comment]
Yep, happy to. Might update the tested versions at the top too, unless you object.

**Pull request merged**